src/App.vue

```diff
@@ -177,11 +179,6 @@ export default {
       return Boolean(sessionStorage.getItem(SessionStorageKeys.KeyCloakToken))
     },
 
-    /** True if Jest is running the code. */
-    isJestRunning (): boolean {
```

Comment:
Moved to getters file.
(We really should create a non-store getters file. Hmm.)

Reply:
Would a mixin work?

Reply:
Yes, good idea.

This has me wondering if we should create "services" modules too (ie, classes that don't depend on Vue).

Next time?

Reply:
Thanks for the feedback. I moved the function to common mixin :)